templates/multisig/contracts/MultisigTemplate.sol

```diff
@@ -7,81 +7,89 @@ import "@aragon/templates-shared/contracts/BaseTemplate.sol";
 contract MultisigTemplate is BaseTemplate {
     using Uint256Helpers for uint256;
 
+    bool constant private TOKEN_TRANSFERABLE = false;
+    uint256 constant private TOKEN_MAX_PER_ACCOUNT = uint256(1);
+
+    string constant private ERROR_EMPTY_SIGNERS = "MULTISIG_EMPTY_SIGNERS";
+    string constant private ERROR_REQUIRED_SIGNATURES_ZERO = "MULTISIG_REQUIRED_SIGNATURE_ZERO";
+    string constant private ERROR_BAD_REQUIRED_SIGNATURES = "MULTISIG_BAD_REQUIRED_SIGNATURES";
+    string constant private ERROR_MISSING_TOKEN_CACHE = "MULTISIG_MISSING_TOKEN_CACHE";
+
     mapping (address => address) internal tokenCache;
 
     constructor(DAOFactory _daoFactory, ENS _ens, MiniMeTokenFactory _miniMeFactory, IFIFSResolvingRegistrar _aragonID)
         BaseTemplate(_daoFactory, _ens, _miniMeFactory, _aragonID)
         public
     {
-        // solium-disable-previous-line no-empty-blocks
+        _ensureAragonIdIsValid(_aragonID);
+        _ensureMiniMeFactoryIsValid(_miniMeFactory);
     }
 
-    function newTokenAndInstance(string tokenName, string tokenSymbol, string id, address[] signers, uint256 requiredSignatures) public {
-        newToken(tokenName, tokenSymbol);
-        newInstance(id, signers, requiredSignatures);
+    function newTokenAndInstance(string _tokenName, string _tokenSymbol, string _id, address[] _signer, uint256 _requiredSignatures) public {
+        newToken(_tokenName, _tokenSymbol);
+        newInstance(_id, _signer, _requiredSignatures);
     }
 
-    function newToken(string name, string symbol) public returns (MiniMeToken) {
-        MiniMeToken token = _createToken(name, symbol);
+    function newToken(string _name, string _symbol) public returns (MiniMeToken) {
+        MiniMeToken token = _createToken(_name, _symbol);
         _cacheToken(token, msg.sender);
         return token;
     }
 
-    function newInstance(string id, address[] signers, uint256 requiredSignatures) public {
-        require(signers.length > 0, "MULTISIG_EMPTY_SIGNERS");
-        require(requiredSignatures > 0, "MULTISIG_REQUIRED_SIGNATURES_ZERO");
-        require(requiredSignatures <= signers.length, "MULTISIG_INVALID_REQUIRED_SIGNATURES");
+    function newInstance(string _id, address[] _signer, uint256 _requiredSignatures) public {
```

Comment:
Nitpick: should we keep the `_signers` variable name here and in `newTokenAndInstance`, since it's an array?

Reply:
Sure, I might have messed it up when renaming the variables